src/openzaak/components/autorisaties/migrations/0005_fill_nlx.py

```diff
@@ -1,44 +1,14 @@
 # SPDX-License-Identifier: EUPL-1.2
 # Copyright (C) 2020 Dimpact
 from django.db import migrations
```

Comment:
Isn't it better if we just detect the Open Zaak version, or see if nlx-url-rewriter is installed, or fail gracefully, in order to prevent the manual intervention? If we can avoid it, we should.

Reply:
We can't really because of the migration dependency which is no longer installed. This graceful approach has been there for a year already.

The point is that I want to remove nlx-url-rewriter, so detecting the installation doesn't make any sense

Reply:
OK.